### ISSUE TITLE
Error building Player: CommandInvokationFailure: Failed to re-package resources. See the Console for details.

### ISSUE THREAD

**Issue by WAZIM414** [googleads/googleads-mobile-unity#295]
Error building Player: CommandInvokationFailure: Failed to re-package resources. See the Console for details.
/Volumes/MacintoshHD2/AndroidSDK/adt-bundle-mac-x86_64-20140702/sdk/build-tools/23.0.1/aapt package --auto-add-overlay -v -f -m -J gen -M AndroidManifest.xml -S "res" -I "/Volumes/MacintoshHD2/AndroidSDK/adt-bundle-mac-x86_64-20140702/sdk/platforms/android-24/android.jar" -F bin/resources.ap_ --extra-packages com.chartboost.sdk.unity:com.google.unity

stderr[
AndroidManifest.xml:20: error: Error: No resource found that matches the given name (at 'value' with value '@integer/google_play_services_version').

AndroidManifest.xml:24: error: Error: No resource found that matches the given name (at 'theme' with value '@style/Theme.IAPTheme').

]
stdout[
Configurations:
 (default)
 ldpi-v4
 hdpi-v4
 xhdpi-v4
 xxhdpi-v4
 xxxhdpi-v4

Files:
  drawable/app_icon.png
    Src: () res/drawable/app_icon.png
    Src: (ldpi-v4) res/drawable-ldpi/app_icon.png
    Src: (hdpi-v4) res/drawable-hdpi/app_icon.png
    Src: (xhdpi-v4) res/drawable-xhdpi/app_icon.png
    Src: (xxhdpi-v4) res/drawable-xxhdpi/app_icon.png
    Src: (xxxhdpi-v4) res/drawable-xxxhdpi/app_icon.png
  values/strings.xml
    Src: () res/values/strings.xml
  AndroidManifest.xml
    Src: () AndroidManifest.xml

Resource Dirs:
  Type drawable
    drawable/app_icon.png
      Src: () res/drawable/app_icon.png
      Src: (ldpi-v4) res/drawable-ldpi/app_icon.png
      Src: (hdpi-v4) res/drawable-hdpi/app_icon.png
      Src: (xhdpi-v4) res/drawable-xhdpi/app_icon.png
      Src: (xxhdpi-v4) res/drawable-xxhdpi/app_icon.png
      Src: (xxxhdpi-v4) res/drawable-xxxhdpi/app_icon.png
  Type values
    values/strings.xml
      Src: () res/values/strings.xml
Including resources from package: /Volumes/MacintoshHD2/AndroidSDK/adt-bundle-mac-x86_64-20140702/sdk/platforms/android-24/android.jar
applyFileOverlay for drawable
applyFileOverlay for layout
applyFileOverlay for anim
applyFileOverlay for animator
applyFileOverlay for interpolator
applyFileOverlay for transition
applyFileOverlay for xml
applyFileOverlay for raw
applyFileOverlay for color
applyFileOverlay for menu
applyFileOverlay for mipmap
Processing image: res/drawable-ldpi/app_icon.png
Processing image: res/drawable/app_icon.png
Processing image: res/drawable-hdpi/app_icon.png
Processing image: res/drawable-xhdpi/app_icon.png
    (processed image res/drawable-ldpi/app_icon.png: 98% size of source)
Processing image: res/drawable-xxhdpi/app_icon.png
    (processed image res/drawable/app_icon.png: 98% size of source)
Processing image: res/drawable-xxxhdpi/app_icon.png
    (processed image res/drawable-hdpi/app_icon.png: 98% size of source)
    (processed image res/drawable-xhdpi/app_icon.png: 98% size of source)
    (processed image res/drawable-xxhdpi/app_icon.png: 98% size of source)
    (processed image res/drawable-xxxhdpi/app_icon.png: 98% size of source)
    (new resource id app_icon from drawable/app_icon.png #generated)
    (new resource id app_icon from ldpi-v4/drawable/app_icon.png #generated)
    (new resource id app_icon from hdpi-v4/drawable/app_icon.png #generated)
    (new resource id app_icon from xhdpi-v4/drawable/app_icon.png #generated)
    (new resource id app_icon from xxhdpi-v4/drawable/app_icon.png #generated)
    (new resource id app_icon from xxxhdpi-v4/drawable/app_icon.png #generated)
]

**Comment by rampara** [comment]
Are you using the latest version of the plugin? Did JarResolver library copy the required Google Play services dependencies into the `Assests/Plugins/Android` directory of your Unity project? If not, can you run the JarResolver library manually by selecting `Assets->Play Services Resolver->Android Resolver->Resolve Client Jars` in the Unity editor.

**Comment by gnchishti** [comment]
I am using GooglePlayGamesPlugin-0.9.34.unitypackage for google play plugin and when I added 
When I added GoogleMobileAds.unitypackage for my google ads, I got same problem.

I dont have Assets->Play Services Resolver->Android Resolver->Resolve Client Jars path rather I have
Assets->Google Play Services->Resolve Client Jars 
and  when I select this option I get this error

FileNotFoundException: Could not find file "/Users/apple/Documents/Projects/Unity/XXX/support-v4-.pom".
System.IO.FileStream..ctor (System.String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, Boolean anonymous, FileOptions options) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.IO/FileStream.cs:305)
System.IO.FileStream..ctor (System.String path, FileMode mode, FileAccess access, FileShare share)
(wrapper remoting-invoke-with-check) System.IO.FileStream:.ctor (string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare)
System.IO.File.OpenRead (System.String path) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.IO/File.cs:363)
System.IO.StreamReader..ctor (System.String path, System.Text.Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.IO/StreamReader.cs:167)
System.IO.StreamReader..ctor (System.String path)
(wrapper remoting-invoke-with-check) System.IO.StreamReader:.ctor (string)
Google.JarResolver.PlayServicesSupport.GetDependencies (Google.JarResolver.Dependency dep)
Google.JarResolver.PlayServicesSupport.ResolveDependencies (Boolean useLatest)
GooglePlayServices.ResolverVer1_1.DoResolutionNoAndroidPackageChecks (Google.JarResolver.PlayServicesSupport svcSupport, System.String destinationDirectory, Google.JarResolver.OverwriteConfirmation handleOverwriteConfirmation) (at Assets/PlayServicesResolver/Editor/ResolverVer1_1.cs:551)
GooglePlayServices.ResolverVer1_1+<DoResolution>c__AnonStorey5.<>m__9 () (at Assets/PlayServicesResolver/Editor/ResolverVer1_1.cs:328)
GooglePlayServices.ResolverVer1_1.DoResolution (Google.JarResolver.PlayServicesSupport svcSupport, System.String destinationDirectory, Google.JarResolver.OverwriteConfirmation handleOverwriteConfirmation, System.Action resolutionComplete) (at Assets/PlayServicesResolver/Editor/ResolverVer1_1.cs:377)
GooglePlayServices.PlayServicesResolver.Resolve (System.Action resolutionComplete) (at Assets/PlayServicesResolver/Editor/PlayServicesResolver.cs:221)
GooglePlayServices.PlayServicesResolver.MenuResolve () (at Assets/PlayServicesResolver/Editor/PlayServicesResolver.cs:244)

any thing I am doing wrong?
